astropy/io/ascii/tests/test_cds_header_from_readme.py

```diff
@@ -6,12 +6,9 @@
 from astropy import units as u
 from astropy.io import ascii
 
-from .common import (  # noqa: F401
-    assert_almost_equal,
-    assert_equal,
-    setup_function,
-    teardown_function,
-)
+from .common import setup_function  # noqa: F401
```

Comment:
tl;dr -- Is this undoing `black` changes?

Reply:
Sorta —  I think more undoing an `isort` change. ``ruff`` sits on top of `isort` and `black`. The former aggregated these imports, the latter separated them over multiple lines. ``ruff`` assumes a project uses both ``isort`` and ``black`` and adds this further style refinement, disaggregating imports with "errors".